k8s-plugin/src/main/kotlin/com/amazon/spinnaker/keel/k8s/resolver/CredentialsResourceHandler.kt

```diff
@@ -82,16 +87,16 @@ class CredentialsResourceHandler(
             SECRET,
             mapOf(
                 "namespace" to resource.spec.namespace,
-                "name" to "${resource.spec.template.metadata["type"]}-${resource.spec.template.data?.account}",
+                "name" to "${resource.spec.template.metadata["type"]}-${clouddriverAccountName}",
                 "annotations" to mapOf("strategy.spinnaker.io/versioned" to "false")
             ),
             null,
-            data
+            data.toMap() as Map<String, Any?>?
         )
     }
 
-    override suspend fun current(resource: Resource<CredentialsResourceSpec>): K8sCredentialManifest? =
-        super.current(resource)?.let {
+    override suspend fun current(r: Resource<CredentialsResourceSpec>): K8sCredentialManifest? =
+        super.current(r)?.let {
```

Comment:
why the change?

Reply:
So the super method's argument is named `r` and I think it's a good practice to match it in case it's called with named argument.

Reply:
in this case since the method is small, it is probably not as bad. Generally though, with bigger methods, the abbreviated form reduces readability. This would have been frowned upon if it was done for a k8s project.